frontend/src/routes/Search.tsx

```diff
@@ -585,7 +599,145 @@ const SearchSeries: React.FC<SearchSeriesProps> = ({ id, title, description, hos
+        },
+        "> div:nth-child(3)": {
+            zIndex: 1,
+            gridColumn: "3 / span 10",
+            gridRow: "1 / span 10",
+        },
+    }}>
+        {
+            thumbnailInfo?.map((info, idx) => <div key={key + idx} css={{
+                outline: `1px solid ${COLORS.neutral10}`,
+                outlineOffset: -.5,
+                borderRadius: 8,
+            }}>
+                <SeriesThumbnail {...{ info, title }} />
+            </div>)
+        }
+    </div>;
+};
+
+type SeriesThumbnailProps = {
+    info: {
+        readonly audioOnly: boolean;
+        readonly isLive: boolean;
+        readonly thumbnail: string | null | undefined;
+    };
+    title: string;
+}
+
+// This is using much of the same styles and logic as the `<Thumbnail>` component for single videos.
+// However, I think the differences are large enough to make modifying the original component to be
+// usable for both cases too much of a hassle. We could consider pulling out some shared css styles,
+// but again, there are subtle differences in most cases.
+const SeriesThumbnail: React.FC<SeriesThumbnailProps> = ({ info, title }) => {
+    const { t } = useTranslation();
+    const isDark = useColorScheme().scheme === "dark";
+
+    let inner;
+    if (info.thumbnail != null) {
+        // We have a proper thumbnail.
+        inner = <ThumbnailImg
+            src={info.thumbnail}
+            alt={t("series.entry-of-series-thumbnail", { series: title })}
+        />;
+    } else {
+        // We have no thumbnail. If the resolution is `null` as well, we are
+        // dealing with an audio-only event and show an appropriate icon.
+        // Otherwise we use a generic icon.
+        const icon = info.audioOnly ? <LuVolume2 /> : <LuFilm />;
+
+        inner = (
+            <div css={{
+                display: "flex",
+                height: "100%",
+                alignItems: "center",
+                justifyContent: "center",
+                fontSize: 40,
+                background: "linear-gradient(135deg, #33333380 50%, transparent 0),"
+                    + "linear-gradient(-135deg, #33333380 50%, transparent 0)",
+                backgroundSize: "17px 17px",
+                color: "#dbdbdb",
+                backgroundColor: "#292929",
+                ...isDark && {
+                    backgroundColor: "#313131",
+                    background: "linear-gradient(135deg, #3e3e3e80 50%, transparent 0),"
+                        + "linear-gradient(-135deg, #3e3e3e80 50%, transparent 0)",
+                },
+            }}>{icon}</div>
+        );
+    }
+
+    const overlay = <div css={{
+        display: "inline-flex",
+        alignItems: "center",
+        gap: 6,
+        position: "absolute",
+        right: 6,
+        bottom: 6,
+        borderRadius: 4,
+        padding: "1px 5px",
+        fontSize: 14,
+        backgroundColor: "rgba(200, 0, 0, 0.9)",
+        color: "white",
+    }}>
```

Comment:
Extracting this into a `ThumbnailOverlay` component with `backgroundColor` and `children` props seem straight forward too. Otherwise it's a bit too much duplication for my taste :/